nextflow_schema.json

```diff
@@ -508,6 +508,52 @@
+                    "description": "Assigns the database /s to be used to annotate the coding regions."
+                },
+                "function_interproscan_enableprecalc": {
+                    "type": "boolean",
+                    "help_text": "This decreases the speed of functional annotation with InterProScan by pre-calculating matches found in the UniProtKB, thereby identifying unique matches in the query sequences for faster annotation. By default this is turned OFF.\n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `---diasable-precalc`",
+                    "description": "Pre-calculates residue mutual matches."
+                },
+                "function_interproscan_enableresidueannot": {
+                    "type": "boolean",
+                    "help_text": "This removes the residue annotation from the final XML and JSON output files, which we do not supply as output files from InerProScan. This is by default turned OFF.\n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `--disable-residue-annot`",
+                    "description": "Enable residue annotation in XML and JSON files."
+                },
+                "function_interproscan_disableresidueannottsv": {
+                    "type": "boolean",
+                    "help_text": "This disables the addition of the annotations assigned according to the databases activated to the final `<sample>_interproscan.tsv` file.  Turning this ON will remove the annotations from the final table. We don't recommend turning this ON to ensure a successful run. \n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `--enable-tsv-residue-annot`\n",
```

Comment:
```suggestion
                    "help_text": "This disables the addition of the annotations assigned according to the databases activated to the final `<sample>_interproscan.tsv` file.  Turning this on will remove the annotations from the final table. It is not recommended to use this option, as it likely will cause a run failure because xxxxxxxxxxxxxxxxxxx. \n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `--enable-tsv-residue-annot`\n",
```